images/cosign/README.md

```diff
@@ -10,13 +10,122 @@
 
 ---
 <!--monopod:end-->
+Minimalist Wolfi-based Cosign images for signing and verifying images using Sigstore.
 
-Minimal image with cosign binary. **EXPERIMENTAL**
+- [Documentation](https://edu.chainguard.dev/chainguard/chainguard-images/reference/cosign)
+- [Provenance Information](https://edu.chainguard.dev/chainguard/chainguard-images/reference/cosign/provenance_info/)
+<!-- TODO: add Getting Started Guide - [Getting Started Guide](https://edu.chainguard.dev/chainguard/chainguard-images/reference/cosign/getting-started-cosign/) -->
 
-## Get It!
+## Image Variants
 
-The image is available on `cgr.dev`:
+Our `latest` tag uses the most recent build of the [Wolfi Cosign](https://github.com/wolfi-dev/os/blob/main/cosign.yaml) package. The following tagged variant is available without authentication:
 
+- `latest`: This is an image for running `cosign` commands. It does not include a shell or other applications.
+
+Starting in April, accessing older tags will require authentication.
```

Comment:
What do you think about adding a year to April (or a full date if we have it), to have a fixed and clear time?

Reply:
I think we should remove this! I actually had it on the template and removed after Adrian's feedback. Marcomm is working on how to better communicate this.

Reply:
Great feedback! I agree with adding a year if we include this warning, but I'll go ahead and remove this line in light of @erikaheidi's comment.

Reply:
Done!